kernel/will-it-scale.py.data/README

```diff
@@ -0,0 +1,79 @@
+Source : https://github.com/antonblanchard/will-it-scale
+
+Will It Scale takes a testcase and runs it from 1 through to n parallel copies
+to see if the testcase will scale. It builds both a process and threads based
+test in order to see any differences between the two.
+
+Following is a list of testcase currently supported.
+
+ - brk1
+ - brk2
+ - context_switch1
+ - dup1
+ - eventfd1
+ - fallocate1
+ - fallocate2
+ - futex1
+ - futex2
+ - futex3
+ - futex4
+ - getppid1
+ - lock1
+ - lock2
+ - lseek1
+ - lseek2
+ - malloc1
+ - malloc2
+ - mmap1
+ - mmap2
+ - open1
+ - open2
+ - open3
+ - page_fault1
+ - page_fault2
+ - page_fault3
+ - pipe1
+ - poll1
+ - poll2
+ - posix_semaphore1
+ - pread1
+ - pread2
+ - pread3
+ - pthread_mutex1
+ - pthread_mutex2
+ - pthread_mutex3
+ - pthread_mutex4
+ - pthread_mutex5
+ - pwrite1
+ - pwrite2
+ - pwrite3
+ - read1
+ - read2
+ - read3
+ - read4
+ - readseek1
+ - readseek2
+ - readseek3
+ - sched_yield
+ - signal1
+ - tlb_flush1
+ - tlb_flush2
+ - tlb_flush3
+ - unix1
+ - unlink1
+ - unlink2
+ - write1
+ - writeseek1
+ - writeseek2
+ - writeseek3
+
+Please refer to the github project documentation on how to add new test cases.
```

Comment:
is this statement relevant to the current repository ? It should  not like 


Please refer to the GitHub project documentation to pick other test cases